packages/ckeditor5-image/tests/image/utils.js

```diff
@@ -890,4 +892,71 @@ describe( 'image widget utils', () => {
 			} );
 		} );
 	} );
+
+	describe( 'getImageModelElementAncestor()', () => {
+		let model;
+
+		beforeEach( async () => {
+			const editor = await VirtualTestEditor.create( {
+				plugins: [ ImageBlockEditing, ImageInlineEditing, Paragraph, ImageCaptionEditing ]
+			} );
+
+			model = editor.model;
+
+			model.schema.register( 'blockWidget', {
+				isObject: true,
+				allowIn: '$root'
+			} );
+
+			editor.conversion.for( 'downcast' ).elementToElement( { model: 'blockWidget', view: 'blockWidget' } );
+		} );
+
+		it( 'should return null if no element is selected and the selection has no image ancestor', () => {
+			setModelData( model, '<paragraph>F[]oo</paragraph>' );
+
+			expect( getImageModelElementAncestor( model.document.selection ) ).to.be.null;
+		} );
+
+		it( 'should return null if a non-image element is selected', () => {
+			setModelData( model, '[<blockWidget></blockWidget>]' );
+
+			expect( getImageModelElementAncestor( model.document.selection ) ).to.be.null;
+		} );
+
+		it( 'should return an imageInline element if it is selected', () => {
+			setModelData( model, '<paragraph>[<imageInline></imageInline>]</paragraph>' );
+
+			const image = getImageModelElementAncestor( model.document.selection );
+
+			expect( image ).to.be.instanceOf( ModelElement );
+			expect( image.name ).to.equal( 'imageInline' );
```

Comment:
`expect( image.is( 'element', 'imageInline' ) ).to.be.true` is enough to check this. The same applies to all tests that follow :point\_down: